src/App.js

```diff
@@ -1,11 +1,24 @@
+import Input from "./Components/Input";
+import Header from "./Components/Header";
+import Button from "./Components/Button";
```

Comment:
if you add a `./Components/index.js` you can re-export all the components e.g.

```js
export { default as Input } from './input`;
```

or you can do named exports from the component modules and then re-exports become even simpler

```js
export * from './input`
```